__tests__/api.test.js

```diff
@@ -78,6 +101,36 @@ describe("GET /api/articles", () => {
         expect(articles).toEqual(dateSortedArticles);
       });
   });
+  describe("ADDITIONAL TEST - GET /api/articles(sorting queries)", () => {
+    it("sorts articles by votes in ascending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes&order=asc")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          const sortedArticles = [...articles].sort((a, b) => a.votes - b.votes);
+          expect(articles).toEqual(sortedArticles)
+        });
```

Comment:
Great way to test it's in order. Could also make use of `jest-sorted -  .toBeSortedBy(votes, {ascending :true})`